go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/polygon-io/errands-server
 
-go 1.12
+go 1.14
```

Comment:
can we go to `1.15`

Reply:
Sure.  We just couldn't stay at 1.12 - it doesn't support numbers of this form: "X_XXX"